pkg/webhook/controlplane/ensurer.go

```diff
@@ -93,6 +93,10 @@ func ensureKubeAPIServerCommandLineArgs(c *corev1.Container) {
 
 	// activate AuthN Webhook with mounted Webhook-Config
 	c.Command = extensionswebhook.EnsureStringWithPrefix(c.Command, "--authentication-token-webhook-config-file=", "/etc/webhook/config/authn-webhook-config.json")
+	// activate dynamic auditing
+	c.Command = extensionswebhook.EnsureStringWithPrefix(c.Command, "--audit-dynamic-configuration=", "true")
+	c.Command = extensionswebhook.EnsureStringWithPrefix(c.Command, "--feature-gates=", "DynamicAuditing=true")
+	c.Command = extensionswebhook.EnsureStringWithPrefix(c.Command, "--runtime-config=", "auditregistration.k8s.io/v1alpha1=true")
```

Comment:
I think this will not a Kubernetes version specific switch, right? I looked it up and it's the same since 1.13?

Reply:
I thought we have to activate this alpha-api? I think all switches are still necessary.
https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#dynamic-backend

Reply:
Right, at least the switch from the controller configuration for enabling the feature would be good.

Reply:
So, leave as is?

Reply:
Surround this by an if-condition that only sets these switches when the splunk webhook logging is enabled.